multichannel_timeseries/anaconda-project.yml

```diff
@@ -0,0 +1,79 @@
+name: multichannel_timeseries
+description: Set of examples to demonstrate the display and annotation of multichannel timeseries data in Neuroscience with HoloViz libraries.
+
+examples_config:
+  created: 2024-08-28
+  maintainers:
+  - "droumis"
+  labels:
+  - "hvplot"
+  - "panel"
+  - "holoviews"
+  categories:
+  - Neuroscience
+  title: "Multichannel Timeseries"
+  deployments:
+  - command: notebook
+  - command: dashboard
+    resource_profile: default
+    auto_deploy: true
+  notebooks_to_skip: []
+  skip_notebooks_evaluation: false
+  no_data_ingestion: false
+  gh_runner: "ubuntu-latest"
+
+user_fields: [examples_config]
+
+channels:
+- conda-forge
+- nodefaults
+
+packages: &pkgs
+- notebook>=6.5.2,<7 # required
+- python=3.11
+- numpy
+- panel>=1.4.2
+- hvplot>=0.10.0
+- pandas>=2.2.1
+- holoviews>=1.19.0 # 1.19: wide df handling, scalebars
+- datashader
+- xarray>=2024.5.0
+- zarr
+- pyarrow
+- dask
+- h5py
+- mne-lsl
+- jupyterlab
+- pip
+- pip:
+  - holonote>=0.2.1 # 0.2.1: visibility and coloring fixes
+  - ndpyramid>=0.2.0
+  - pyproj==3.6.1
+  - tsdownsample>=0.1.3
+  - bokeh>=3.6.0 # 3.6.0: scalebar on subplots
+  - wget
+  - mne[hdf5]
+
+dependencies: *pkgs
+
+commands:
+  notebook:
+    notebook: index.ipynb
+  dashboard:
+    unix: panel serve --rest-session-info --session-history -1 index.ipynb
```

Comment:
Do you really want to deploy `index.ipynb`? As far as I can see it doesn't serve any Panel app (but it can be deployed as a dashboard thanks to this new Panel feature that allows to deploy notebooks without any `.servable()` calls). You could:
- deploy nothing :)
- deploy one specific notebook
- deploy multiple notebooks

Reply:
good catch! I guess I'll serve just the one recommended workflow multichan.ipynb